service/templates/base.html

```diff
@@ -71,7 +71,7 @@
                     <span>Skip to main content</span>
                 </a>
                 <a class="logo" href="//jisc.ac.uk">
-                    <img alt="Jisc logo" src="{{url_for('static', filename='images/jisc-logo.png')}}">
+                     <img alt="Jisc logo" width="43" height="25" src="{{url_for('static', filename='images/jisc-logo.png')}}">
```

Comment:
Is this the correct way of doing said image? Should original be resized, or do sizes belong here or in CSS?
@TomYeahOK - thoughts?